src/field_theory/polynomial_galois_group.lean

```diff
@@ -195,7 +200,7 @@ end roots_action
 
 variables {p q}
 
-/-- The restriction homomorphism between Galois groups -/
+/-- `polynomial.gal.restrict`, specialized to when both fields are extensions of polynomials. -/
```

Comment:
Should `extensions of polynomials` be changed to `splitting fields of polynomials`?

Reply:
Or maybe something like: "`polynomial.gal.restrict` for divisible polynomials." (analogous to the docstring in the composition case)